sync/sync_test.go

```diff
@@ -169,16 +223,21 @@ func TestSync_WithMarathonProblems(t *testing.T) {
 }
 
 func TestSync_WithConsulProblems(t *testing.T) {
+	t.Parallel()
 	// given
 	marathon := marathon.MarathonerStubForApps()
 	consul := errorConsul{}
-	sync := New(marathon, consul)
+	sync := newSync(marathon, consul)
 	// when
 	err := sync.SyncServices()
 	// then
 	assert.Error(t, err)
 }
 
+func newSync(marathon marathon.Marathoner, service consul.ConsulServices) *Sync {
```

Comment:
I'm not sure if this name is good. What's difference between `newSync` and `New`? `newSync` just call `New` with default config, name should represent it. How about `NewWithDefaultConfig`?